src/index.html

```diff
@@ -4,10 +4,303 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="icon" href="./images/favicon.png">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
     <script type="text/javascript" src="scripts/main.js"></script>
+
+    <header class="header">
+      <div class="header__phone-logo">
+        <a href="tel:+1 234 555-55-55"><img src="./images/Vector (1).png" alt="phone-logo"></a>
+      </div>
+      <input
+          type="checkbox"
+          id="burger"
+          class="header__burger"
+      >
```

Comment:
завеликі відступи для атрибутів
```suggestion
      <input
        type="checkbox"
        id="burger"
        class="header__burger"
      >
```